### ISSUE TITLE
Feature request: A better waypoint system and pod

### ISSUE THREAD

**Issue by ZHANGTIANYAO1** [Project-Hatchet/public_h-60#371]
## **Describe feature request.**
Some ideas that may able to improve the waypoint system and pod usage experience
1. Waypoint information can be synchronized with the MicroDAGR
2. "quick waypoint" can work again(shift+left mouse bottom on the map can create a "quick waypoint" and can directly attach the pod to it, which works before 0.6)
3. Maybe some features from the Turret Enhanced mod(Slew turret to typed coordinates, Add Black, Blue or Red numbered markers on the map where the turret is pointing, Slew turret to position on map via click etc.)
4. Moving Target Indicator(I think this feature also works before 0.6 but seems been removed after the update)

**Comment by FireStone1029** [comment]
the nav system / page that we currently have is somthing that we are not happy with and is on the list of reworks, so any suggesting on how to improve it are always helpful can i ask what you mean by
"microdagr synchrinized" & "moving target indicator"

Also you currently can slew the flir to a waypoint / pos on the map you have to do it though the tac map's menu

**Comment by ZHANGTIANYAO1** [comment]
"microdagr synchrinized": means when I change, add, or remove waypoints from MicroDAGR I do not need to import them again into the helicopter in order to update the info. (I saw the reply from discord, yes maybe just creating another way to modify the waypoint could be better e.g. A Tactical tablet? )

 "moving target indicator": Automatic tracking of moving vehicles like cars, trucks, tanks, or something else

**Comment by FireStone1029** [comment]
sorry you would like point tracking to work on the flir as it tracking a moving car you you want a waypoint that will move with a car?

**Comment by ZHANGTIANYAO1** [comment]
> sorry you would like point tracking to work on the flir as it tracking a moving car you you want a waypoint that will move with a car?

not really, I don't want a waypoint that will move with a car but just FLIR can tracking a moving car

**Comment by FireStone1029** [comment]
that is currently a bug that we are looking at fixing
